20-ConfirmNavigation/src/PrismSample/Views/ViewAPage.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PrismSample.Views.ViewAPage"
```

Comment:
Use a meaningful name for the page

Reply:
That will not be easy 🤣 but I will change this. Thank you.